packages/lesswrong/server/languageModels/postSummaryResolver.ts

```diff
@@ -0,0 +1,27 @@
+import { Posts } from '../../lib/collections/posts/collection';
+import { augmentFieldsDict } from '../../lib/utils/schemaUtils'
+import { dataToMarkdown } from '../editor/conversionUtils';
+import { languageModelGenerateText } from './languageModelIntegration';
+
+augmentFieldsDict(Posts, {
+  languageModelSummary: {
+    resolveAs: {
+      type: "String",
+      resolver: async (post: DbPost, args: void, context: ResolverContext): Promise<string> => {
+        const markdownPostBody = dataToMarkdown(post.contents?.originalContents?.data, post.contents?.originalContents?.type);
+        const authorName = "Authorname"; //TODO
```

Comment:
TODO?

Reply:
This is in the admin-gated auto-summarizer (I built on that branch because it has other stuff related to language model access); it doesn't get called by autotagging.

Reply:
Ok, may want to note somewhere the current state and usability